src/osrm/osrm.cpp

```diff
@@ -44,26 +43,12 @@ OSRM::OSRM(engine::EngineConfig &config)
     // Now, check that the algorithm requested can be used with the data
     // that's available.
 
-    if (config.algorithm == EngineConfig::Algorithm::CoreCH ||
-        config.algorithm == EngineConfig::Algorithm::CH)
+    if (config.algorithm == EngineConfig::Algorithm::CH)
```

Comment:
This also needs to check for `config.algorithm == EngineConfig::Algorithm::CoreCH` since we can't remove the `CoreCH` value yet. Doing so we be a breaking change of the library interface.